app/lib/account/agent.dart

```diff
@@ -18,6 +18,11 @@ abstract class KnownAgents {
   // Agent for pub admin actions.
```

Comment:
Since:
```
  static const githubActions = 'service:github-actions';
  static const gcpServiceAccount = 'service:gcp-service-account';
```

are no longer just static values, but also prefixes. I think it's critically important that they each have documentation comment explaining.
 * Summary line what they are
 * Description of the form the take, e.g. `service:github-actions:<org>/<repository>`.
 * Description of the form they used to have `service:github-actions`.
 * If we support both forms for new entries, then we critically need to document when and why each of the two forms are used. As suggested further down, I propose we only allow the form `service:github-actions:<org>/<repository>` for new actions.

Reply:
renamed + added documentation. PTAL.